tests/globalhelper/helper.go

```diff
@@ -361,3 +363,19 @@ func CopyFiles(src string, dst string) error {
 
 	return nil
 }
+
+var seededRand *rand.Rand = rand.New(
+	rand.NewSource(time.Now().UnixNano()))
+
+func GenerateRandomString(length int) string {
+	charset := "abcdefghijklmnopqrstuvwxyz"
+
+	//nolint:varnamelen
```

Comment:
same comments as in https://github.com/test-network-function/cnfcert-tests-verification/pull/446
why not pull all these utility function in a new package and add new ones for before and after each?